lib/option.rb

```diff
@@ -27,8 +28,11 @@ module Option
     end
   end
 
+  AI_MODELS = ai_models.select { _1["enabled"] }
+
   PROVIDERS.freeze
   LOCATIONS.freeze
+  AI_MODELS.freeze
```

Comment:
Since `freeze` returns self, you could condense this into one line:  `ai_models.select { _1["enabled"] }.freeze`